index.bs

```diff
@@ -2737,13 +2737,15 @@ PDF document represented as a Base64-encoded string.
         params: browsingContext.PrintParameters
       }
 
+      browsingContext.PageRange = text .regexp "^(?:[0-9]+)?(?:-(?:[0-9]+)?)?$"
```

Comment:
Is the intent here just to limit what constitutes a valid value for the text the user sends, if they choose to send text as the argument for `pageRanges`?

Reply:
Yes. For example, "5-3" would be matched however it's not valid (AFAIUI). I think @jrandolf is only interested in the overall text format, not necessarily in the full validation.

Still - see comment above - "`'-'" is matched but it should not be allowed.

Reply:
Indeed.

Reply:
Fair enough. Nothing to do for local ends/clients, then. This is all on the remote-end implementation.